### PR TITLE
Prepare v0.2.2 release

### PR DESCRIPTION
**Pull request by drybjed**

### CODE REVIEW

**Comment by drybjed** [comment]
@ypid Can you check this PR real quick? I want to revamp the network configuration in the role so that it uses merged dictionaries a'la `debops.ifupdown`, so I think that releasing the current changes now as `v0.2.2` should be fine, and the next release can be `v0.3.0` with more significant changes.

**Review by ypid — CHANGES_REQUESTED** [review]
@drybjed Sounds good. But according to [SemVer](http://semver.org/spec/v2.0.0.html), the release needs to be v0.3.0 as there are breaking changes.

**Comment by drybjed** [comment]
@ypid OK then, I'll update the version.

File: docs/scripts/upgrade-from-v0.2.X-to-v0.3.X [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-## Upgrade inventory variables for migration from debops.owncloud v0.2.X to v0.2.2.
+## Upgrade inventory variables for migration from debops.owncloud v0.2.X to v0.3.X.
```
Comment (ypid) [inline docs/scripts/upgrade-from-v0.2.X-to-v0.3.X]: `debops.owncloud` my damn copy/paste behavior :wink: Could you fix that for me?

**Review by ypid — APPROVED** [review]

**Comment by drybjed** [comment]
@ypid I guess that we're both blind. :-)

**Pull request merged**